server/webhook_worker.go

```diff
@@ -56,15 +56,14 @@ func (ww webhookWorker) process(msg *webhookMessage) (err error) {
 	if err != nil {
 		return err
 	}
-
-	if _, _, err = wh.PostNotifications(ww.p, msg.InstanceID); err != nil {
-		ww.p.errorf("WebhookWorker id: %d, error posting notifications, err: %v", ww.id, err)
-	}
-
 	v := wh.(*webhook)
 	if err = v.JiraWebhook.expandIssue(ww.p, msg.InstanceID); err != nil {
 		return err
 	}
+	err = wh.(*webhook).applyReporterNotification(msg.Data, v.Issue.Fields.Reporter)
+	if _, _, err = wh.PostNotifications(ww.p, msg.InstanceID); err != nil {
+		ww.p.errorf("WebhookWorker id: %d, error posting notifications, err: %v", ww.id, err)
+	}
```

Comment:
Can we add a newline in between each of these separate operations? Also I don't think `applyReporterNotification` needs to return an error. See my comments on that function above

```suggestion
	}

	v.applyReporterNotification(msg.Data, v.Issue.Fields.Reporter)

	if _, _, err = wh.PostNotifications(ww.p, msg.InstanceID); err != nil {
		ww.p.errorf("WebhookWorker id: %d, error posting notifications, err: %v", ww.id, err)
	}
```

Reply:
Why do we need to pass in `msg.Data` here? All of the webhook data has already been unmarshaled into the `webhook` struct

Reply:
my bad , have fixed the broken piece of code